sixtyfps_compiler/build.rs

```diff
@@ -36,7 +43,13 @@ fn widget_library() -> &'static [(&'static str, &'static BuiltinDirectory<'stati
         )?;
     }
 
-    writeln!(file, "]\n}}")?;
+    writeln!(file, "]\n}}\n\n")?;
+
+    writeln!(
+        file,
+        "fn builtin_styles() -> Vec<&'static str> {{ vec![\"{}\"] }}\n",
```

Comment:
Shouldn't this perhaps simply be a `&'static [&'static str]` instead of a newly allocated vector every time? :)